packages/@tinacms/cli/src/cmds/statusChecks/waitForIndexing.ts

```diff
@@ -34,27 +37,23 @@ class IndexFailedError extends Error {
 }
 
 export const waitForDB = async (ctx, next, options: { verbose?: boolean }) => {
-  if (options.verbose) {
-    logger.info(logText('Waiting for DB...'))
-  }
-
   if (!ctx.client) {
-    if (options.verbose) {
-      logger.info(logText('client is unavailable, skipping...'))
-    }
-    return next()
+    throw new Error(
+      'No Tina Cloud find. For more information on how to setup the tina cloud see https://tina.io/docs/features/data-fetching/#making-requests-with-the-tina-client'
```

Comment:
```suggestion
      'No Tina Cloud found. For more information on how to setup the tina cloud see https://tina.io/docs/features/data-fetching/#making-requests-with-the-tina-client'
```

Though I'm not sure this error makes a lot of sense in this context. Is this happening when we don't pass in a `client`?

Reply:
Yeah, I think this might happen if the user does not provide a client ID, token and branch.